src/RestApi/StoreApi/Schemas/OrderSchema.php

```diff
@@ -0,0 +1,625 @@
+							'readonly'    => true,
+						],
+						'total_fees_tax'     => [
+							'description' => __( 'Total tax on fees.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+						'total_discount'     => [
+							'description' => __( 'Total discount from applied coupons.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+						'total_discount_tax' => [
+							'description' => __( 'Total tax removed due to discount from applied coupons.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+						'total_shipping'     => [
+							'description' => __( 'Total price of shipping.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+						'total_shipping_tax' => [
+							'description' => __( 'Total tax on shipping.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+						'total_price'        => [
+							'description' => __( 'Total price the customer will pay.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+						'total_tax'          => [
+							'description' => __( 'Total tax applied to items and shipping.', 'woo-gutenberg-products-block' ),
+							'type'        => 'string',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+						],
+						'tax_lines'          => [
+							'description' => __( 'Lines of taxes applied to items and shipping.', 'woo-gutenberg-products-block' ),
+							'type'        => 'array',
+							'context'     => [ 'view', 'edit' ],
+							'readonly'    => true,
+							'items'       => [
+								'type'       => 'object',
+								'properties' => [
+									'name'  => [
+										'description' => __( 'The name of the tax.', 'woo-gutenberg-products-block' ),
+										'type'        => 'string',
+										'context'     => [ 'view', 'edit' ],
+										'readonly'    => true,
+									],
+									'price' => [
+										'description' => __( 'The amount of tax charged.', 'woo-gutenberg-products-block' ),
+										'type'        => 'string',
+										'context'     => [ 'view', 'edit' ],
+										'readonly'    => true,
+									],
+								],
+							],
+						],
+					]
+				),
+			],
+		];
+	}
+
+	/**
+	 * Convert a woo order into an object suitable for the response.
+	 *
+	 * @param \WC_Order $object Order class instance.
+	 * @return array
+	 */
+	public function get_item_response( $object ) {
+		$order_item_schema = new OrderItemSchema();
+
```

Comment:
Should validate `$object` is an instance of `\WC_Order` before using it?  I realize we can't typehint here because this is an implementation of the abstract class method.

Reply:
We can here, it's not in the abstract class.